rclpy/rclpy/node.py

```diff
@@ -756,6 +756,22 @@ def _set_parameters_atomically(
             self._parameter_event_publisher.publish(parameter_event)
 
         return result
+    
+    def add_on_set_paramters_callback(self,callback:Callable([List[Parameter],SetParametersResult])):
```

Comment:
Please run linter tests, it will catch errors like this, also, I think you need to use square brackets (`[` `]`) in type annotations  instead of parentheses:
```suggestion
    def add_on_set_paramters_callback(self, callback: Callable[[List[Parameter], SetParametersResult]]):
```